app/scripts/metamask-controller.js

```diff
@@ -6652,12 +6652,31 @@ export default class MetamaskController extends EventEmitter {
    * @param {string} origin - the domain to safelist
    */
   safelistPhishingDomain(origin) {
+    this.metaMetricsController.trackEvent({
+      category: MetaMetricsEventCategory.Phishing,
+      event: MetaMetricsEventName.ProceedAnywayPhishingPage,
+      properties: {
+        origin,
+      },
+    });
+
     return this.phishingController.bypass(origin);
   }
 
   async backToSafetyPhishingWarning() {
-    const extensionURL = this.platform.getExtensionURL();
-    await this.platform.switchToAnotherURL(undefined, extensionURL);
+    const portfolioURL =
```

Comment:
We have a `PORTFOLIO_URL` env variable.. we probably should use that instead.
It would look something like:
```js
`${process.env.PORTFOLIO_URL}/?metamaskEntry=phishing_page_portfolio_button&marketingEnabled=true`
```